### PR TITLE
add Chinese (zh_CN) translations to 1.14.7, full coverage

### PR DESCRIPTION
**Pull request by slightlyskepticalpotat**
@patricklodder is this what you meant when you suggested that I just include your commits in the translation PR?

### CODE REVIEW

**Comment by patricklodder** [comment]
Yes, perfect! I'm just going to mark this as blocked until the string freeze gets merged.

**Review by patricklodder — CHANGES_REQUESTED** [review]
The linter caught 3 errors that would lead to runtime errors for users with this language selection. Thank you for helping me prove that the linter works ❤️

I've marked each in-line.

**Comment by slightlyskepticalpotat** [comment]
Does the linter check the entire file at once? I fixed those issues and a few more popped up (it seems some are from previous contributions, and others are from me).

**Comment by patricklodder** [comment]
> Does the linter check the entire file at once?

Yes. I just pseudo-fixed them myself locally and I get this after:

```console
$ python3 contrib/devtools/check-translations.py 
Checking all translations...
No errors detected.
```

File: src/qt/locale/bitcoin_zh_CN.ts
Comment (patricklodder) [inline src/qt/locale/bitcoin_zh_CN.ts:1589]: changes source - this means this line in the translation will not be applied

Comment (slightlyskepticalpotat) [reply src/qt/locale/bitcoin_zh_CN.ts]: Apologies, should be fixed now—at least we know that the linter works :)

**Review by patricklodder — COMMENTED** [review]
code review ACK. Passes linter!

Could use review by a (native) speaker, I have tagged "help wanted" for this.

File: src/qt/locale/bitcoin_zh_CN.ts
Comment (MarsDoge) [inline src/qt/locale/bitcoin_zh_CN.ts:1431]: 可以用你备份的密钥来**恢复**钱包和狗狗币

Comment (MarsDoge) [inline src/qt/locale/bitcoin_zh_CN.ts:1431]: 可以用你备份的密钥来收复钱包和狗狗币 -> 可以用你备份的密钥来**恢复**钱包和狗狗币

File: src/qt/locale/bitcoin_zh_CN.ts
Comment (MarsDoge) [inline src/qt/locale/bitcoin_zh_CN.ts:1463]: 核心repository -> 核心**仓库**

**Review by MarsDoge — CHANGES_REQUESTED** [review]

File: src/qt/locale/bitcoin_zh_CN.ts
Comment (slightlyskepticalpotat) [reply src/qt/locale/bitcoin_zh_CN.ts:1463]: I think 狗狗币核心repository is more suited as it is "狗狗币核心 repository" and not "狗狗币 核心仓库".

File: src/qt/locale/bitcoin_zh_CN.ts
Comment (slightlyskepticalpotat) [reply src/qt/locale/bitcoin_zh_CN.ts:1431]: Will fix, thanks for catching this!

File: src/qt/locale/bitcoin_zh_CN.ts
Comment (MarsDoge) [reply src/qt/locale/bitcoin_zh_CN.ts:1463]: okay

**Comment by patricklodder** [comment]
@slightlyskepticalpotat for some reason, GitHub is not processing the update to 1.14.7 - it should either stop recognizing my commits in your branch or throw a fit about merge conflicts, but it does neither. 
I'm not sure why.

Would you mind rebasing with just your latest commit off of 1.14.7-dev (ad6a62b7) so that this can go into approval?

**Comment by slightlyskepticalpotat** [comment]
@patricklodder I decided to try this before rebasing, looks like it worked.
Screenshot_20240218_074122_Chrome

**Comment by patricklodder** [comment]
>  I decided to try this before rebasing, looks like it worked.

Awesome hack 😂

**Review by patricklodder — APPROVED** [review]
ACK - code reviewed and tested on x86_64 Ubuntu Jammy

Language validated by @MarsDoge - many thanks!

**Review by Jody78 — COMMENTED** [review]
Thank you

**Pull request merged**
